### DISCUSSION TITLE
Find the odd one

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#966] [Galaxy Trivia] [answered]
**ICGARRAE**
**GUMNTEAR**
**OUETHOUS**
**ETENNTEM**

**Comment by peterbaccaro** [comment, marked as answer]
1. CARRIAGE
2. ARGUMENT
3. OUTHOUSE
4. TENEMENT

The odd one out in this list is **ARGUMENT** (i.e. GUMNTEAR).

CARRIAGE, OUTHOUSE, and TENEMENT are all related to physical structures or objects. A carriage refers to a horse-drawn vehicle, an outhouse refers to an outdoor toilet, and a tenement refers to a multi-occupancy building, usually in an urban area.

On the other hand, ARGUMENT is not related to physical structures or objects. It refers to a verbal dispute or disagreement between people.
